tethysapp/quest/controllers/quest_workflows.py

```diff
@@ -143,10 +144,33 @@ def new_collection_workflow(request):
     collection_description = request.POST.get('collection_description', "")
     collection = utilities.generate_new_collection(collection_name,
                                                    collection_description)
+    try:
+        result = get_collection_html(request, collection)
+        result['success'] = True
+        alert_context = {
+            'alert_style': 'success',
+            'alert_message': 'The collection was successfully created.'
+        }
+        alert_html = render(request, 'quest/alert.html', alert_context).content.decode('utf-8')
+    # Added code here to alert user if collection was not created successfully
+    except ValueError as e:
+        print(e)
```

Comment:
Is this print statement suppose to be here? I didn't see it anywhere else in the workflows and was just curious. Thanks